frontend/src/components/profile-happening-preview.tsx

```diff
@@ -0,0 +1,103 @@
+                        <Box>
+                            <Stack textAlign="right">
+                                <Flex alignItems="center" justifyContent="flex-end">
+                                    <BiCalendar />
+                                    {isToday(new Date(event.date)) ? (
+                                        <>
+                                            <Text ml="1" fontWeight="bold">
+                                                {isNorwegian ? `I dag ` : `Today `}
+                                            </Text>
+                                            <Text ml="1" fontSize="1rem">
+                                                {format(new Date(event.date), isNorwegian ? 'HH:mm' : 'h:aaa')}
+                                            </Text>
+                                        </>
+                                    ) : (
+                                        format(new Date(event.date), 'dd. MMM', { locale: isNorwegian ? nb : enUS })
+                                    )}
+                                </Flex>
+                            </Stack>
+                        </Box>
+                    </Flex>
+                </LinkBox>
+            ) : (
+                <LinkBox data-testid={event.slug}>
+                    <Flex alignItems="center">
+                        <Box>
+                            <LinkOverlay
+                                as={NextLink}
+                                href={`/event/${event.slug}`}
+                                _hover={{ textDecor: 'underline' }}
+                            >
+                                {event.title}
+                            </LinkOverlay>
+                        </Box>
+
+                        <Spacer />
+
+                        <Box>
+                            <Stack textAlign="right">
+                                <Flex alignItems="center" justifyContent="flex-end">
+                                    <BiCalendar />
+                                    {isToday(new Date(event.date)) ? (
+                                        <>
+                                            <Text ml="1" fontWeight="bold">
+                                                {isNorwegian ? `I dag ` : `Today `}
+                                            </Text>
+                                            <Text ml="1" fontSize="1rem">
+                                                {format(new Date(event.date), isNorwegian ? 'HH:mm' : 'h:aaa')}
+                                            </Text>
+                                        </>
+                                    ) : (
+                                        format(new Date(event.date), 'dd. MMM', { locale: isNorwegian ? nb : enUS })
+                                    )}
+                                </Flex>
+                            </Stack>
+                        </Box>
+                    </Flex>
+                </LinkBox>
+            )}
```

Comment:
Koden på de to forskjellige er jo veeeldig lik. Den kan nok kombineres ganske lett og heller skille på bedpres og arrangement kun der dere trenger det.